file1.py

```diff
@@ -0,0 +1,41 @@
+from typing import Union
+
+from fastapi import FastAPI
+
+
+
+app = FastAPI()
+
+
+@app.get("/")
+def read_root():
+    return {"Hello": "World"}
+
+
+@app.get("/items/{item_id}")
+def read_item(item_id: int, q: Union[str, None] = None):
+    return {"item_id": item_id, "q": q}
+    
+
+class BankAccount():
+
+    def __init__(self, account_id, account_user):
+        self.account_id = account_id
+        self.account_user = account_user
+
+    def get_user(self):
+        return self.account_user
+
+    def set_user(self, new_user):
+        self.account_user = new_user
+
+    def get_id(self):
+        return self.account_id
+
+    def set_id(self, new_id):
+        self.account_id = new_id
```

Comment:
would be nice to move this to its own module

Reply:
sure